x-pack/plugins/observability_solution/observability/server/services/slo/slo_settings.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsClientContract } from '@kbn/core-saved-objects-api-server';
+import { ElasticsearchClient } from '@kbn/core-elasticsearch-server';
+import { SLO_SUMMARY_DESTINATION_INDEX_PATTERN } from '../../../common/slo/constants';
+import { SloSettings } from '../../domain/models';
+import { sloSettingsObjectId, SO_SLO_SETTINGS_TYPE } from '../../saved_objects/slo_settings';
+
+export const getSloSettings = async (soClient: SavedObjectsClientContract) => {
+  try {
+    const object = await soClient.get<SloSettings>(SO_SLO_SETTINGS_TYPE, sloSettingsObjectId);
+    return object.attributes;
+  } catch (e) {
+    return {
```

Comment:
Log this error. Also, do we need to bubble this error up to the user?

Reply:
We might need to discriminate on a specific error, return the empty object, and bubble up all the other errors.